### PR TITLE
Remove deprecated OC_Helper::linkTo

### PR DESCRIPTION
**Pull request by rullzer**
More code cleanup.

All calls to the function are replaced with the suggested body of the function.

CC: @MorrisJobke @PVince81 @LukasReschke

### CODE REVIEW

**Comment by MorrisJobke** [comment]
Tested and works :+1:

**Comment by LukasReschke** [comment]
:+1:

File: lib/private/template/functions.php [hunk 1]
```diff
@@ -146,7 +146,7 @@ function component($app, $file) {
  * For further information have a look at OC_Helper::linkTo
```
Comment (mmattel) [inline lib/private/template/functions.php]: Should´nt PHPDoc be updated too?

Comment (MorrisJobke) [inline lib/private/template/functions.php]: Nice catch

**Pull request merged**